error-prone-contrib/src/main/java/tech/picnic/errorprone/refastertemplates/ReactorTemplates.java

```diff
@@ -271,6 +271,32 @@ Flux<S> after(Flux<T> flux) {
     }
   }
 
+  /** Prefer {@link Flux#onErrorComplete()} over more contrived alternatives */
+  static final class FluxOnErrorComplete<T> {
```

Comment:
Although the added templates are sorted lexicographical, I see that within this class we usually list the `Mono` before the `Flux`, so I'm swapping them around.